docs/development.md

```diff
@@ -44,6 +44,10 @@ Run the following command:
 ```sh
 ./gradlew appengineRun
 ```
+Run the command below instead if you want the console output to be visible: 
+```sh
+./gradlew appengineRun -Pdisable_daemon
+```
 
 Wait until the task exits with a `BUILD SUCCESSFUL`.
```

Comment:
This instruction only applies to the original command. Move it to the first instruction on line 42:
``Run the following command and wait until the task exits with a `BUILD SUCCESSFUL`:``